<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Flight</class>
 <widget class="QDialog" name="Add_Flight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Flight</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 37, 41);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 18pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>Add Flight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Source Airport:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Destination Airport: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="sourceComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="destinationComboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 240, 245);
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 240, 245);
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Distance:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="distanceLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
